starter-code/javascript/chronometer.js

```diff
@@ -1,27 +1,32 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
-
-// Chronometer.prototype.setTime = function () {
-
-// };
+ function Chronometer() {
+  this.currentTime=0;
+  this.intervalId=0;
+}
+
+Chronometer.prototype.startClick = function () {
+  this.intervalId = setInterval(function() {
```

Comment:
Niiiiice!

Reply:
Maybe here you loose the context, so you have to define above ```var self = this``` and then inside the interval you do ```self.currentTime++```